oci/layout/oci_dest.go

```diff
@@ -127,13 +128,13 @@ func createManifest(m []byte) ([]byte, string, error) {
 		}
 		return b, om.MediaType, nil
 	case manifest.DockerV2ListMediaType:
-		return nil, "", fmt.Errorf("can't create OCI manifest from Docker V2 schema 2 manifest list")
+		return nil, "", errors.New("can't create an OCI manifest from Docker V2 schema 2 manifest list")
 	case imgspecv1.MediaTypeImageManifestList:
-		return nil, "", fmt.Errorf("can't create OCI manifest from OCI manifest list")
+		return nil, "", errors.New("can't create an OCI manifest from OCI manifest list")
 	case imgspecv1.MediaTypeImageManifest:
 		return m, mt, nil
 	}
-	return nil, "", fmt.Errorf("Unrecognized manifest media type: %q", mt)
+	return nil, "", errors.New("unrecognized manifest media type")
```

Comment:
mm what does it mean to "quote" the type? the only thing we can get from GuessMIMEType and switch here https://github.com/containers/image/pull/84/files#diff-c61203eb5d39307008ae5e34098ad9fbR109 is an empty string if the manifest mime type is unrecognized - is there any helpfulness in quoting the empty string?